cli/command/image/build_test.go

```diff
@@ -58,6 +60,7 @@ func TestRunBuildResetsUidAndGidInContext(t *testing.T) {
 	}
 }
 func TestRunBuildDockerfileFromStdinWithCompress(t *testing.T) {
+	skip.IfCondition(t, runtime.GOOS == "openbsd", "not working right on openbsd")
```

Comment:
Could you elaborate on what didn't work so that we have a better idea of when this skip can be removed?

Reply:
See above. Seems to be a permissions issue running as an unprivileged user. This also happens if doing `make test` on macOS. 

I'd say these aren't specific to OpenBSD then and really an issue of being able to run the unit tests without elevated privileges.

Reply:
Yup, sounds right. Thanks for looking into it. 

Would you mind changing the skip condition to be `os.Getuid() != 0` that way it's clear why the test is being skipped.